hw_diag/utilities/sentry.py

```diff
@@ -0,0 +1,31 @@
+import logging
+
+import sentry_sdk
+from hm_pyhelper.util.sentry import before_send_filter
+from sentry_sdk.integrations.flask import FlaskIntegration
+from sentry_sdk.integrations.logging import LoggingIntegration
+
+
+def init_sentry(sentry_dsn, release, balena_id, balena_app):
+    """
+    Initialize sentry with balena_id and balena_app as tag.
+    If sentry_dsn is not set, do nothing.
+    """
+
+    if not sentry_dsn:
+        return
+
+    sentry_logging = LoggingIntegration(
+        level=logging.CRITICAL,
```

Comment:
we can go back to error logging. We had decided sometime back that rate limiting is better than silencing errors level

Reply:
I didn't change the log level in this PR. We can change the level in another PR.

Reply:
Or we can do it here.

Reply:
Updated!